src/containers/Tenant/Diagnostics/Overview/TopicStats/TopicStats.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../../styles/mixins.scss';
+
+.ydb-overview-topic-stats {
+    &__title {
+        @include info-viewer-title();
+    }
+
+    .ydb-loader {
+        margin-top: 50px;
+    }
+
+    .ydb-bars {
+        margin-top: -5px;
+    }
+
+    &__info {
+        .info-viewer__label-text {
+            display: flex;
+            overflow: visible;
+
+            max-width: 150px;
+
+            white-space: normal;
+        }
+    }
+
+    &__bytes-written {
+        margin-top: 7px;
+
+        .info-viewer {
+            padding-left: 20px;
```

Comment:
this is however _is_ a reference to another block, which could be a style for an element of the current block